src/Core/Form/IdentifiableObject/CommandBuilder/Product/PricesCommandsBuilder.php

```diff
@@ -30,12 +30,23 @@
 
 use PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductPricesCommand;
 use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopConstraint;
 
 /**
  * Builder used to build UpdateProductPricesCommand
  */
-class PricesCommandsBuilder implements ProductCommandsBuilderInterface
+class PricesCommandsBuilder extends MultistoreCommandsBuilder implements ProductCommandsBuilderInterface
 {
+    /**
+     * @var UpdateProductPricesCommand
+     */
+    private $singleShopCommand;
```

Comment:
Why keep the two separate? We will never have a part of the form submitted for single shop and another part submitted for all shops?